terraform/dashboard/alb.tf

```diff
@@ -29,17 +29,27 @@ resource "aws_lb_listener_rule" "service_listener_rule" {
     }
   }
 
-  dynamic "action" {
-    for_each = var.authenticate_users ? [1] : []
+  # dynamic "action" {
```

Comment:
I think the idea of the dynamic action was to allow you to enable the auth by setting the authenticate_users variable to 1.  Did you try this?  If you are not using the dynamic element, it'd be best delete the commented code and run terraform fmt to ensure the code is formatted correctly.

Reply:
Keeping to static action to match Repo Archive Tool